docs/products/compute/compute-instances/guides/lish/index.md

```diff
@@ -209,127 +198,223 @@ If you are having issues accessing Lish, Weblish, or Glish, you may be blocked b
 - **Glish port:** 8080
 {{< /note >}}
 
-### Lish Gateway Fingerprints
+#### Atlanta, GA (USA)
 
-The valid fingerprints for the Lish gateway boxes are as follows:
+-   **Lish SSH Gateway:** `lish-atlanta.linode.com`
 
-#### Atlanta
+    {{< note type="secondary" title="Lish SSH Gateway Fingerprints" isCollapsible=true >}}
+    ```command
+    RSA 59:30:1a:0b:93:5e:3f:4b:6f:d1:96:ff:7e:9e:12:f8
+    ECDSA SHA256:8emv5PuUgPB2GFejMYWl1f4x1yj3YqAQPAYIrBm43ZI
+    Ed25519 SHA256:7k2c442k+zqbGaraZvmqXM3MA5lCcthaR2lbrB651lg
+    Ed25519 MD5:b3:76:56:3a:d8:66:5e:0c:61:90:03:b7:0b:ca:58:25
+    ```
+    {{< /note >}}
 
-These are the fingerprints for the Lish gateway in our Atlanta data center (lish-atlanta.linode.com):
+-   **Weblish Gateway:** `atlanta.webconsole.linode.com`
+-   **Glish Gateway:**
 
-```command
-RSA 59:30:1a:0b:93:5e:3f:4b:6f:d1:96:ff:7e:9e:12:f8
-ECDSA SHA256:8emv5PuUgPB2GFejMYWl1f4x1yj3YqAQPAYIrBm43ZI
-Ed25519 SHA256:7k2c442k+zqbGaraZvmqXM3MA5lCcthaR2lbrB651lg
-Ed25519 MD5:b3:76:56:3a:d8:66:5e:0c:61:90:03:b7:0b:ca:58:25
-```
+#### Chicago, IL (USA)
 
-#### Dallas
+-   **Lish SSH Gateway:** `lish-us-ord.linode.com`
 
-These are the fingerprints for the Lish gateway in our Dallas data center (lish-dallas.linode.com):
+    {{< note type="secondary" title="Lish SSH Gateway Fingerprints" isCollapsible=true >}}
+    ```command
+    RSA 3072 SHA256:rRwktOKfSApeffa+YOVxXXL70Ba1CpTYp/oFywEH2Pc lish-us-ord.linode.com
+    ECDSA 256 SHA256:SV9A/24Jdb++ns/+6Gx7WqZCyN4+0y4ICFsaqK3Rm8s lish-us-ord.linode.com
+    ED25519 256 SHA256:J+yN8rjhr9j27M4zLSF6OX9XmIoipWbPP/J1AGRlRYc lish-us-ord.linode.com
+    ```
+    {{< /note >}}
 
-```command
-RSA 6d:3d:b5:d0:42:1c:49:45:a6:47:29:bd:88:4e:58:d4
-ECDSA SHA256:1fL1HTGas1APIpshCu1ZWys7LI97s8eTFN3+f8zEYXA
-Ed25519 SHA256:Ime9h7IAxAjBvMGR+G+EnbaLWpvXO+Z7TaGHzM9g5Sc
-Ed25519 MD5:23:c1:02:ba:40:77:f7:73:0b:d4:0f:3a:f1:8c:74:b6
-```
+-   **Weblish Gateway:** `us-ord.webconsole.linode.com`
+-   **Glish Gateway:** `ord2.glish.linode.com`
 
-#### Frankfurt
+#### Dallas, TX (USA)
 
-These are the fingerprints for the Lish gateway in our Frankfurt data center (lish-frankfurt.linode.com):
+-   **Lish SSH Gateway:** `lish-dallas.linode.com`
 
-```command
-RSA 43:76:22:43:0e:01:cb:84:6a:80:b9:9b:90:34:c7:b1
-ECDSA SHA256:e1FxEXiZVi6n13tagd1ZAQEW/fsRqz29ez5IfWf9kxg
-Ed25519 SHA256:vG1rnoGe7XRRY0nauJREQk75OamxCwRRpeaTDB8LpgM
-Ed25519 MD5:9e:83:86:e2:f9:f7:f7:56:fc:bf:54:bb:75:7e:79:37
-```
+    {{< note type="secondary" title="Lish SSH Gateway Fingerprints" isCollapsible=true >}}
+    ```command
+    RSA 6d:3d:b5:d0:42:1c:49:45:a6:47:29:bd:88:4e:58:d4
+    ECDSA SHA256:1fL1HTGas1APIpshCu1ZWys7LI97s8eTFN3+f8zEYXA
+    Ed25519 SHA256:Ime9h7IAxAjBvMGR+G+EnbaLWpvXO+Z7TaGHzM9g5Sc
+    Ed25519 MD5:23:c1:02:ba:40:77:f7:73:0b:d4:0f:3a:f1:8c:74:b6
+    ```
+    {{< /note >}}
 
-#### Fremont
+-   **Weblish Gateway:** `dallas.webconsole.linode.com`
+-   **Glish Gateway:**
```

Comment:
missing glish gateway for a number of DCs

Reply:
I added these back in. Thanks!